github-adapter/src/main/java/be/xplore/githubmetrics/githubadapter/RepositoriesAdapter.java

```diff
@@ -29,8 +30,10 @@ public RepositoriesAdapter(GithubConfig config, GithubAdapter githubAdapter) {
         this.githubAdapter = githubAdapter;
     }
 
+    @Cacheable("Repositories")
     @Override
     public List<Repository> getAllRepositories() {
+        LOGGER.error("this is getting un cached response");
```

Comment:
I assume this is a todo that still needs resolving?

Reply:
removed